src/sentry/static/sentry/app/views/onboarding/projectSetup/inviteMembers.jsx

```diff
@@ -0,0 +1,129 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {addSuccessMessage} from 'app/actionCreators/indicator';
+import {t, tct} from 'app/locale';
+import Alert from 'app/components/alert';
+import EmailField from 'app/views/settings/components/forms/emailField';
+import Form from 'app/views/settings/components/forms/form';
+import HookOrDefault from 'app/components/hookOrDefault';
+import Panel from 'app/components/panels/panel';
+import SelectField from 'app/views/settings/components/forms/selectField';
+import SentryTypes from 'app/sentryTypes';
+import TextBlock from 'app/views/settings/components/text/textBlock';
+import space from 'app/styles/space';
+import withApi from 'app/utils/withApi';
+import withConfig from 'app/utils/withConfig';
+
+class InviteMembers extends React.Component {
+  static propTypes = {
+    api: PropTypes.object.isRequired,
+    orgId: PropTypes.string.isRequired,
+    project: SentryTypes.Project.isRequired,
+    config: SentryTypes.Config.isRequired,
+  };
+
+  state = {
+    invitedEmails: [],
+    roleList: [],
+  };
+
+  componentDidMount() {
+    this.fetchRoleDetails();
+  }
+
+  async fetchRoleDetails() {
+    const {api, orgId} = this.props;
+
+    const member = await api.requestPromise(`/organizations/${orgId}/members/me/`);
```

Comment:
Can we move this to a util function and out of the component?

Reply:
1fbc172